catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -102,6 +102,8 @@
 
     public static final String CACHED = "cached";
 
+    private static final String INTERNAL_LOCAL_RESOURCE = "internal.local-resource";
```

Comment:
@figliold and I talked about this but it's a little tricky in this case. The attribute is added by the plugin, so it should own the constant. However, since it is a plugin, the endpoints that check for it (currently the search endpoint) cannot assume the plugin has been deployed and therefore can't have a dependency on it.

Any other suggestions?